app/controller/sdl/Abstract/Controller.js

```diff
@@ -92,6 +92,14 @@ SDL.SDLController = Em.Object.extend(
             );
             break;
           }
+          case -4:
```

Comment:
@mkorniichuk as a part of this PR it would be nice to add a new enum into `data.js` like for a `resultCode` and use a named enum values instead of these magic numbers...

Reply:
@AKalinich-Luxoft good point. Fixed in b12ddf49de672d428b21918946a12149f87384f9